IPython/core/display.py

```diff
@@ -869,14 +870,14 @@ def _repr_html_(self):
             mimetype, encoding = mimetypes.guess_type(self.filename)
 
             video = open(self.filename, 'rb').read()
-            video_encoded = video.encode('base64')
+            video_encoded = base64.b64encode(video)
         else:
             video_encoded = self.data
             mimetype = self.mimetype
         output = """<video controls>
  <source src="data:{0};base64,{1}" type="{0}">
  Your browser does not support the video tag.
- </video>""".format(mimetype, video_encoded)
+ </video>""".format(mimetype, video_encoded.decode('ASCII'))
```

Comment:
I think that makes sense, thanks.

I don't know enough about video formats to know what `_repr_?_`, what mimetype etc. would make sense for video data - I guess there are several different formats that might be used. So I think it's simplest to just stick with `_repr_html_` for now.